profiler/README.md

```diff
@@ -0,0 +1,107 @@
+to analyze the collected profiles. For the Go language, your analysis options include
+CPU time, heap, allocated heap, contention, and thread analysis. You can even
+compare sets of profiles.
+
+For profiling concepts, the capabilities of the Profiler UI, and how to configure your application
+to collect and transmit profiling data, see the
+[Profiler documentation](https://cloud.google.com/profiler/docs).
+
+See the following configuration guides for details on modifying your application to collect
+and transmit profiling data:
+
++ [Profiling Go applications](https://cloud.google.com/profiler/docs/profiling-go)
++ [Profiling Java applications](https://cloud.google.com/profiler/docs/profiling-java)
++ [Profiling Node.js applications](https://cloud.google.com/profiler/docs/profiling-nodejs)
++ [Profiling Python applications](https://cloud.google.com/profiler/docs/profiling-python)
+
+See
+[Profiling applications runing outside of Google Cloud Platform](https://cloud.google.com/profiler/docs/profiling-external) 
+for additional configuration steps that are required when you are running your service outside of
+Google Cloud Platform.
+
+## Samples
+
+Detailed instructions on executing these samples from the Cloud Shell is included in
+[Profiling samples](https://cloud.google.com/profiler/docs/samples).
+
+### profiler_quickstart
+
+The sample `profiler_quickstart` is configured to run the `hello-profiler` service.
+This is a very simple service that is used by the
+[Quickstart](https://cloud.google.com/profiler/docs/quickstart) guide.
+
+[Go Code](/profiler/profiler_quickstart)
+
+### hotapp
+
+The sample `hotapp` is uses an infinite loop with two call stacks.
+
+The [Profiler documentation](https://cloud.google.com/profiler/docs)
+includes images generated from this sample. 
+If you wish to generate profile data consistent with that included in the Profiler documentation,
+run the `hotapp` service with the following command line options:
+```
+       go run main.go -service=docdemo-service -local_work -skew=75 -version=1.75.0
+```
+
+[Go Code](/profiler/hotapp)
+
+### hotmid
+
+Sample `hotmid` is an application that simulates multiple calls to a library
+function made via different call paths. Each of these calls is not
+particularly expensive (and so does not stand out on the flame graph). But
+in the aggregate these calls add up to a significant time which can be
+identified via looking at the flat list of functions' self and total time.
+
+[Go Code](/profiler/hotmid)
+
+## Executing a sample
+
+To execute a sample and collect profiling data in your GCP project, do the following:
+
+1.  If you have a new GCP, you need to enable the Profiler API for your project. Choose one of the following methods.
+
+    From the Cloud console, go to **APIs & Services** and then click **Enable APIS and Services**.
+    Search for **Profiler**.  If the API isn't enabled, click **Enable**.
+
+    From the Cloud Shell, run the following command:
+    
+```
```

Comment:
Indent the whole backtick block, there shouldn't be added indent for the code.

Reply:
Changes made. Thanks.